README.md

```diff
@@ -73,7 +73,7 @@ To run the packaged version of the app e.g. for macOS run:
 
 The app is configured for mainnet by default but you can opt-in to testnet:
 ```
-./dist/mac/Lightning.app/Contents/MacOS/Lightning --bitcoin.testnet --bitcoin.node=neutrino --neutrino.connect=btcd-testnet.lightning.computer --neutrino.feeurl=https://nodes.lightning.computer/fees/v1/btctestnet-fee-estimates.json
+./dist/mac/Lightning.app/Contents/MacOS/Lightning --bitcoin.testnet --bitcoin.node=neutrino --neutrino.connect=btcd-testnet.lightning.computer --neutrino.feeurl=https://nodes.lightning.computer/fees/v1/btc-fee-estimates.json
```

Comment:
The testnet one only supports 2 confs:
````json
{
  "fee_by_block_target": {
    "2": 1012
  }
}
````

cc @alexbosworth

Reply:
Sorry @tanx for not explaining this, if there is only 2 confs listed it means all other fees are equal. You can use the testnet fee endpoint, it will change if there are more fee levels detected by Bitcoin Core

Here is a doc that explains it https://github.com/lightninglabs/app-api#fee-estimates